### PR TITLE
second commit of new branch

### PR DESCRIPTION
**Pull request by AzimsHub**

### CODE REVIEW

**Review by Alpenglow1988 — CHANGES_REQUESTED** [review]
Empty file?

**Review by AzimsHub — COMMENTED** [review]
Yes empty

File: test.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+hello
+this
+is
+a
+
+
```
Comment (Alpenglow1988) [inline test.txt]: This is badly formatted! please change this

File: test.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+bunch
+
+of new
+
+lines 
+
+
+bad
```
Comment (Alpenglow1988) [inline test.txt]: Word broken over lines

File: test.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+hello
+this
+is
+a
+
+
+bunch
+
+of new
```
Comment (Alpenglow1988) [inline test.txt]: Phrasing

**Review by Alpenglow1988 — CHANGES_REQUESTED** [review]

File: test.txt (hunk 1 above)
Comment (AzimsHub) [reply test.txt]: fixed

File: test.txt (hunk 2 above)
Comment (AzimsHub) [reply test.txt]: fixed

File: test.txt (hunk 3 above)
Comment (AzimsHub) [reply test.txt]: fixed

**Review by Alpenglow1988 — APPROVED** [review]
LGTM! 👍🏽

**Pull request merged**